protocol/testing/containertest/testnet_test.go

```diff
@@ -80,6 +84,9 @@ func expectProto(t *testing.T, tag string, message proto.Message) bool {
 }
 
 func TestPlaceOrder(t *testing.T) {
+	if os.Getenv("SKIP_DISABLED") != "" {
```

Comment:
Can we add some documentation around why these specific tests are disabled? Doesn't seem like this specific to upgrade testing?